Felienne/hedy
# Adds status banner when having a pending teacher request

**Description**
In this PR we add a banner when a user has a pending teacher account request. This way they know that the request has not been handled yet. The banner is removed again once the request is accepted / denied by the admin.

**Fixes**
This PR fixed #3177.

**How to test**
Make sure you have an account with a teacher request. Navigate to your profile page and notice the new status banner shown at the top of the screen. When accepting the request as an admin -> the banner should disappear.

**Screenshot**
![Schermafbeelding 2022-08-18 om 14 14 11](https://user-images.githubusercontent.com/38155424/185392086-612ebd5b-077a-40ad-a642-7569d6fb8a24.png)
